.dockerignore

```diff
@@ -0,0 +1,15 @@
+#Git
+.git
+.gitignore
+
+#Docker
+.Docker

```

Comment:
Docker already ignores files starting with a dot, as well as .git and .docker directories by default.